config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
-  root to: 'tasks#index'
-  resources :labels
-  resources :users
+   namespace :admin do
+    resources :users
+  end
   resources :tasks
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root to: 'sessions#new'
+  resources :users, only: [:new, :create, :show]
```

Comment:
`%i`記法を使うとスペース区切りで簡潔に記載できます。
```suggestion
  resources :users, only: %i[new create show]
```